org.mozilla.FirefoxNightly/org.mozilla.FirefoxNightly.json

```diff
@@ -160,8 +177,12 @@
                   "path": "mozlog-no-blessings-req.patch"
               },
               {
-                "type": "patch",
-                "path": "firefox-pipewire.patch"
+                  "type": "patch",
+                  "path": "firefox-pipewire.patch"
+              },
+              {
+                  "type": "patch",
+                  "path": "firefox-extension-dir.patch"
```

Comment:
I wouldn't take this patch approach. See https://bugzilla.mozilla.org/show_bug.cgi?id=1352416 - upstream prefer that the same Linux binary works inside and outside a flatpak. The easiest is probably simply to add to the finish-args something like:
`--env=MOZILLA_PLUGIN_PATH=/app/lib/mozilla/plugins`

This will then be set in the environment when Firefox is executed.

Reply:
Thanks for pointing that out, I've looked into the https://bugzilla.mozilla.org/show_bug.cgi?id=1332404#c12. Looks like you mean `MOZ_PLUGIN_PATH`, that should work for us.